upi/openstack/inventory.yaml

```diff
@@ -32,3 +28,16 @@ all:
       # Number of provisioned Compute nodes.
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
+
+      # If set to 'true', the 'bootstrap' playbook will create a floating IP
+      # attached to the bootstrap node, to enable the Installer to collect a
+      # 'must-gather' log bundle, useful for debugging a failed installation.
+      os_bootstrap_fip: true
```

Comment:
this is an extra variable. can't we ust "when os_api_fip is defined condition"?  in the playbooks